### ISSUE TITLE
Intellisense out of sync after project-level search and replace

### ISSUE THREAD

**Issue by leocb** [microsoft/vscode-cpptools#4085]
**Type: LanguageService**

**Describe the bug**
- OS and Version: Windows 10 Home - Build 17763
- VS Code Version: 1.37.0 (system)
- C/C++ Extension Version: 0.24.1
- Other extensions: Clang-format - Yes the bug persists

After using the Search and replace (workspace-wide), the Intellisense cannot parse the new field, throwing red squiggles at me (x has no field y)
I can confirm it happens with any struct, but I don't know if it affects other kinds of code (like enums or defines)
The only way I found to solve this is by Reloading VSCode, then it detects the new field name correctly
Sometimes it does work correctly, but it's far more common for me to get the errors.

**To Reproduce**
1. Search for a field name that exists inside a struct
2. Replace it with something else
3. See error

**Expected behavior**
after replacing the field name, the Intellisense should update accordingly

**Screenshots**
Before replace:
![image](https://user-images.githubusercontent.com/8310271/63032539-6292d680-be8c-11e9-908d-b21ac406b4d9.png)

After replace:
![image](https://user-images.githubusercontent.com/8310271/63032481-498a2580-be8c-11e9-822f-163e6e27969f.png)

**Additional context**
- This is a STM32 C project
- `c_cpp_properties.json`:
``` json
{
    "configurations": [{
        "name": "STM32 Debug",
        "includePath": [
            "${env:VSARM}/arm-none-eabi/include",
            "${env:VSARM}/arm-none-eabi/include/c++/8.2.1",
            "${env:VSARM}/arm-none-eabi/include/c++/8.2.1/backward",
            "${env:VSARM}/arm-none-eabi/include/c++/8.2.1/arm-none-eabi",
            "${env:VSARM}/lib/gcc/arm-none-eabi/8.2.1/include",
            "${env:VSARM}/lib/gcc/arm-none-eabi/8.2.1/include-fixed",
            "${workspaceRoot}/Inc",
            "${workspaceRoot}/Src",
            "${workspaceRoot}/Src/tasks",
            "${workspaceRoot}/Src/trucktag",
            "${workspaceRoot}/Libs",
            "${workspaceRoot}/Libs/spiffs",
            "${workspaceRoot}/Libs/utilities",
            "${workspaceRoot}/Drivers/STM32F1xx_HAL_Driver/Inc",
            "${workspaceRoot}/Drivers/STM32F1xx_HAL_Driver/Inc/Legacy",
            "${workspaceRoot}/Middlewares/Third_Party/FreeRTOS/Source/include",
            "${workspaceRoot}/Middlewares/Third_Party/FreeRTOS/Source/CMSIS_RTOS",
            "${workspaceRoot}/Middlewares/Third_Party/FreeRTOS/Source/portable/GCC/ARM_CM3",
            "${workspaceRoot}/Drivers/CMSIS/Device/ST/STM32F1xx/Include",
            "${workspaceRoot}/Drivers/CMSIS/Include",
            "${workspaceRoot}/Middlewares/ST/STM32_USB_Device_Library/Core/Inc",
            "${workspaceRoot}/Middlewares/ST/STM32_USB_Device_Library/Class/CDC/Inc"
        ],
        "defines": [
            "DEBUG",
            "STM32F103xE",
            "USE_HAL_DRIVER",
            "uint32_t=unsigned int",
            "__SCHAR_MAX__=0x7f",
            "__SHRT_MAX__=0x7fff"
        ],
        "intelliSenseMode": "clang-x64",
        "browse": {
            "path": [
                "${workspaceRoot}",
                "${env:VSARM}"
            ],
            "limitSymbolsToIncludedHeaders": false,
            "databaseFilename": ""
        }
    }],
    "version": 4
}
```

**Comment by bobbrow** [comment]
May be related to #3883

**Comment by bobbrow** [comment]
This appears to be fixed in 1.2.0.  Please let us know if you still have any issues with this.  Thank you!
